game/Env.py

```diff
@@ -46,7 +46,7 @@ def check_user_events(self):
                 self.game.close_board()
 
 
-    def game_state(self, col_choice: int) -> str:
+    def assign_rewards(self, col_choice: int) -> str:
         '''
         Returns the game state seen for the perspective for self.currentPlayer.
```

Comment:
This docstring is no longer valid. Do you care to just write a single sentence so we avoid confusing ourselves later on?